doc/non-ansible-configuration-notes.md

```diff
@@ -216,3 +216,95 @@ Also ensure that /usr/local/bin is included in the `JENKINS_PATH` set in /etc/in
 [`ansible.intro_windows`]: http://docs.ansible.com/ansible/intro_windows.html
 [newer Ansible configuration]: https://github.com/nodejs/build/tree/master/ansible
 [stand-alone]: https://github.com/nodejs/build/tree/master/setup/windows
+
+## AIX 7.2
+
+To set up, basically:
+
+1. Run ansible
+2. Run through ansible/aix61-standalone/manualBootstrap.md, doing the steps marked as
```

Comment:
That file doesn't exist!

Reply:
does now thanks to #1620
https://github.com/nodejs/build/blob/master/ansible/aix61-standalone/manualBootstrap.md